contracts/IClaimable.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.6.12;
+pragma solidity >=0.6.12 <=0.8.0;
```

Comment:
Doesn't this rule out 0.8.1 (and more generally, the latest working version of 0.8.x)?

You probably want to change that to either `<=0.7.6` or `<0.8.0` (or if you do want to include 0.8.x, then  `<0.9.0`).

Reply:
changed it to `pragma solidity >=0.6.12 <=0.7.6;`